Source/ACE/Entity/PhysicsData.cs

```diff
@@ -30,10 +30,10 @@ public class PhysicsData
         public float Friction;
         public float Elastcity;
         public uint AnimationFrame;
-        public Position Acceleration;
+        public AceVector3 Acceleration;
```

Comment:
System.Numerics.vector3 does not write packet payloads in the correct order.
But I will add the ability to return a vector.

But AceVector follows the position class, although position is over loaded